README.rst

```diff
@@ -27,7 +27,7 @@ Status
 ------
 
 This project's status is experimental. It has been tested with CircuitPython (SAMD) boards and the
-Feather HUZZAH (ESP8266) with CircuitPython 3.x.
+Feather HUZZAH (ESP8266) with CircuitPython 3.x and Raspberry Pi Pico CircuitPython 6.x.
```

Comment:
Could you put this on a new line?

Reply:
Yes will do,Thank you very much for the comments, I will review this tonight